dist/cms.js

```diff
@@ -35,10 +35,52 @@ function getWordPressEmbedBlockCounts() {
   }
 }
 
+/**
+ * Obtains data about scripts that WordPress prints on the page.
+ *
+ * @returns {{}[]}
+ */
+function getWordPressScripts() {
+  const entries = [];
+
+  /**
+   * Checks whether the provided element is an inline script.
+   *
+   * @param {?Element} element Element to examine. May be null.
+   * @return {?number} Size of the inline script or null if the element isn't an inline script.
+   */
+  const getInlineScriptSize = (element) => {
+    if (element instanceof HTMLScriptElement && !element.src) {
+      return element.textContent.length;
+    }
+    return null;
+  };
+
+  const scripts = document.querySelectorAll('script[src][id$="-js"]');
+  for (const script of scripts) {
+    /** @var HTMLScriptElement script */
+    const handle = script.id.replace(/-js$/, '');
+    entries.push({
+      handle,
+      src: script.src,
+      in_footer: script.parentNode !== document.head,
+      async: script.async,
+      defer: script.defer,
+      intended_strategy: script.dataset.wpStrategy || null,
+      after_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-after`)),
+      before_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-before`)),
+      extra_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-extra`)),
+      translations_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-translations`)),
```

Comment:
I think it's worth adding a comment here as I was a bit confused here.

```suggestion
      // For each external script, check if there is a related inline script
      after_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-after`)),
      before_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-before`)),
      extra_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-extra`)),
      translations_script_size: getInlineScriptSize(document.getElementById(`${handle}-js-translations`)),
```